core/src/main/java/hudson/model/View.java

```diff
@@ -552,18 +549,11 @@ public String getSearchUrl() {
     	List<Action> result = new ArrayList<Action>();
     	result.addAll(getOwner().getViewActions());
     	synchronized (this) {
```

Comment:
@agentgonzo I don't think there is a need to keep this `synchronized` anymore is there?

Reply:
nope